CMakeLists.txt

```diff
@@ -23,7 +23,8 @@ include(CheckCXXCompilerFlag)
 set_policies(
   TYPE NEW
   POLICIES CMP0073
-           CMP0074)
+           CMP0074
+           CMP0079)
```

Comment:
Does this by any chance bump up minimum required cmake version to 3.13 where this policy is introduced to fix behavior of `target_link_libraries` command ?

Reply:
Policies are compatible with older versions.